src/FileIO.scala

```diff
@@ -0,0 +1,50 @@
+import java.io.{File, PrintWriter}
+
+import scala.collection.mutable
+import scala.io.Source
+/**
+  * Furniture, Alcohol, Toiletries, Shoes, Food, Jewelry
+  * customer_id,date,credit_card,cvv,category
+  * Created by jakefroeb on 10/18/16.
+  */
+object FileIO {
+  def main(args: Array[String]): Unit = {
+    val purchases = mutable.MutableList[(String, String, String, String, String)]()
+    Source
+      .fromFile("purchases.csv")
+      .getLines.drop(1)
+      .foreach(line => {
+        val Array(customerId, date, creditCard, cvv, category) = line.split(",").map(_.trim)
+        purchases.+=((customerId, date, creditCard, cvv, category))
+      })
+
+    def prompt(s: String) = {
+      println(s);
+      io.StdIn.readLine()
+    }
+    def sort(s: String) = {
+      val pw = new PrintWriter(new File(s"filtered${s}purchases.prn") )
+      purchases.foreach(x =>
+        if ((x._5).equalsIgnoreCase(s)) {
```

Comment:
is it possible to filter the purchases using a functional approach?  Hint: ".filter()"